frontend/src/pages/AgreementPage.tsx

```diff
@@ -79,10 +124,36 @@ const AgreementPage: FC = () => {
             <Typography variant="body1"> View Agreement</Typography>
           </Button>
         </Stack>
-        <Typography sx={{ paddingLeft: '3rem', fontSize: '16px' }} variant="body2">
+        <Typography sx={{ paddingLeft: '3rem', fontSize: '1rem' }} variant="body2">
           {agreementContext.rationale || 'rationale'}
         </Typography>
       </Stack>
+      <Stack direction="column">
+        {Object.keys(agreemetChaptersData).map((chapterName: string, i: number) => {
+          const chapter = agreemetChaptersData[chapterName];
+          return (
+            <Stack direction="column" key={i}>
+              <Stack direction="row" alignItems="center" height="4.5rem" columnGap="1.5rem">
+                <Typography fontWeight="500" fontSize="1.1rem">
+                  # {chapterName}
+                </Typography>
+                <Typography fontSize="0.87rem" fontWeight="noraml" color="text.secondary">
+                  {Object.keys(chapter).length} sections
+                </Typography>
+                <Typography fontSize="0.87rem" fontWeight="noraml" color="text.secondary">
+                  {calcChapterSuggestions(chapter)} suggestions
+                </Typography>
+              </Stack>
+              <Stack direction="column" rowGap="2rem" width="70%">
```

Comment:
No need for width here. Section cards should take available space.
Instead - The Stack containing all of the chapters should use `maxWidth="md"`